src/serializers/transactions/transaction-details-serializer.ts

```diff
@@ -19,6 +20,7 @@ const transactionDetailsSerializer = (transaction: Transaction, user: User): Tra
     createdAt: transaction.created_at,
     state: transaction.state,
     avatarUrl: user.avatar_url,
+    total_amount: transaction.total_amount,
```

Comment:
on frontend we use camelCase so lines in the serializer need to be like `totalAmount: transaction.total_amount`